.github/workflows/send-redirects-to-production.yml

```diff
@@ -0,0 +1,56 @@
+
+# Terraform Action to lint and apply updated redirects
+name: Publish Production Content and Redirects
+on:
+  workflow_dispatch:
+jobs:
+  publish-to-prod:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: scripts/redirect_creation
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+
+      - name: create Terraform Tfvars
+        id: tfvars
+        run: |
+          cat << EOF > terraform.auto.tfvars 
+            stage                 = "prod"
+          EOF
+
+      - name: Terraform Format
+        id: fmt
+        run: terraform fmt
+
+      - name: Terraform Init
+        id: init
+        run: terraform init
+
+      - name: Terraform Validate
+        id: validate
+        run: terraform validate -no-color
+
+      - name: Terraform Plan
```

Comment:
I'd like to see the plan show up in the PR.  Here's a good tutorial:
https://developer.hashicorp.com/terraform/tutorials/automation/github-actions